ckanext/datagovuk/tests/test_inventory.py

```diff
@@ -0,0 +1,62 @@
+import os
+import uuid
+
+import ckan.plugins.toolkit as toolkit
+import ckan.tests.factories as factories
+from ckan.model import Package
+from ckanext.datagovuk.harvesters.inventory_harvester import InventoryHarvester
+from ckanext.harvest.model import setup as db_setup
+from ckanext.harvest.model import (HarvestObject, HarvestGatherError,
+                                   HarvestObjectError, HarvestSource,
+                                   HarvestJob)
+
+class TestInventory:
+
+    def setup(self):
+        db_setup()
+
+        self.sysadmin = factories.Sysadmin()
+        self.publisher = factories.Organization()
+        self.source = HarvestSource(
```

Comment:
This should really be created with harvest_source_create action, but there are problems identifying the inventory harvester as it assumes the BaseHarvester.